src/cmd/select.rs

```diff
@@ -21,6 +21,20 @@ selected using regular expressions.
   Select the third column named 'Foo':
   $ qsv select 'Foo[2]'
 
+  Select the first and last columns, 9999 is a special index for the last column:
+  $ qsv select 1,9999
+
+  Reverse the order of columns:
+  $ qsv select 9999-1
+
+  Sort the columns lexicographically. Note that you must provide a dummy selector:
```

Comment:
As a user, I would expect sorting/shuffling to work simultaneously, i.e. it would select specified columns, then sort/shuffle if `--sort` and/or `--random` is provided.

This way `qsv select 1 --sort` would be meaningless, as sorting just a single column is trivial.